src/Subscription/SubscriptionPreview/SubscriptionPreview.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
If you put both preview project and GA project to master,  they will be released together.

Reply:
We are adding just GA project now